docs/GOVERNANCE.md

```diff
@@ -159,51 +165,79 @@ The specifications produced will adhere to the following:
   that the "minor" version number will always be zero and the `specversion`
   string will always be of the form `X.0`.
 
-Note that these rules do not apply to the
+Note that these rules do not apply to unversioned documents, such as the
 [documented extensions](../cloudevents/extensions/README.md).
 
-All versions are tagged from the `main` branch, but the tag only applies to
-the "subject" of the release - the directory containing the information
-covered by that release (e.g. `subscriptions` or `cloudevents`). The
-[CloudEvents web site](https://cloudevents.io/) takes appropriate content from
-each tagged version. (If the directory containing the information covered
-by the release is not in a top-level directory, the subject should be the full path,
-e.g. `top-dir/sub-dir`.)
-
-> Note: should the need arise, additional branches may be created. For example,
-> it is likely that a `core-v2.0` branch will be created to collect changes for
-> the core specification version 2.0 significantly before those changes are
-> merged into the main branch, to allow for ongoing work on the main branch.
-> Such branches should be deleted once their content is eventually merged.
-
 To create a new release:
 
 - Periodically the group will examine the list of extensions to determine
   if any action should be taken (e.g. removed due to it being stale). The
   creation of a new release will be the reminder to do this check. If any
   changes are needed then PRs will be created and reviewed by the group.
-- Create a PR that modifies the [README](README.md), and all specifications (ie.
-  \*.md files) that include a version string, to the new release version string.
-  Make sure to remove `-wip` from all of the version strings.
+
+- For the most part we try to only use a single branch ("main") for our work.
+  Git branches will be used as a way to snapshot/tag releases. While we hope
+  to not push new commits to these branches, in practice, it is possible that
+  changes to a release might be needed without "picking up" other changes that
+  might have been commited to "main". Creating the branch during the release
+  process will makes this easier to manage should the need should arise.
+
+- Determine the new release version string. It should be of the form:
+  `<subject>/vX.Y.Z`, e.g. `cloudevents/v1.0.4` or `subscriptions/v1.0.0`.
+
+- Create a PR (for the "main" branch") that:
+  - Modifies the repo's files to use the new version string appended with
+    `-rc#` as appropriate. Make sure to remove all `-wip` suffixes as needed.
+  - Merge the PR.
+  - Initiate a final review/test of the release.
+
+- A "release candidate" tag will be created with the new release version
```

Comment:
I thought we weren't going to use tags at all, on the grounds that we'd need to keep the tag up-to-date with the branch?

Reply:
yea, I'm not thrilled with the idea of dual maintenance once an RC is created. If the RC lives for a long time that's just going to be annoying. If you think having an RC tag is bad, or confusing, then I'm ok with just telling people to look at the tip of "main" for their review.